docs/Containers/Container-Tasks.md

```diff
@@ -62,50 +62,66 @@ the initial release for container tasks.
+security requirements, Evergreen does not support using arbitrary images for container tasks, and the custom
+Dockerfile you use must adhere to these [requirements](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit).
 At the moment, our image security automation is still [under
 development](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit#heading=h.ghiwibz2opmd),
-so for the time being, modifying a project\'s respective Dockerfile
-requires the submission of a PR against our repo.
-
-1.  Stage a PR in our centralized image
-    [repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
-    This repo hosts Dockerfiles for all teams participating in the
-    beta of running container tasks in Evergreen. The PR, when merged
-    into the main branch, triggers an automated Drone pipeline which
-    then builds the image and pushes it to Amazon Elastic Container
-    Registry (ECR). To onboard, your PR changes must include a
-    relevant Dockerfile, created under a new directory that has the
-    name of your project. Review should be requested from
+so for the time being, you may create your own Dockerfile for use in container tasks but R&D Dev Prod will be temporarily
+housing your Dockerfile in a managed [repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
+
+
+### Creating And Managing Your Container Images
+
+The following is the process by which you can create a usable image from your custom Dockerfile:
+
+1.  Ensure you are properly permissioned to stage PRs in the centralized image
+    repository, which hosts Dockerfiles for all teams participating in the
+    experimental offering of running container tasks in Evergreen.
+    Permission can be requested from MANA [here](https://mana.corp.mongodbgov.com/resources/64c464185f1589fbe7faec38).
+2.  Stage a PR against the repo containing a relevant Dockerfile under a new directory
+    that has the name of your project. Review should be requested from
     **evergreen-ci/evg-app**, and the PR's approval will be contingent
     on the Dockerfile following the [secure image
     policy](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit)
     that we currently have in place.
-
-2.  Once the PR has been approved and is merged into main with the Drone
-    build passing, you may reference the URI of the newly built image.
+3.  The PR, when approved and merged into the main branch, will trigger an automated Drone pipeline which
+    builds the image and pushes it to Amazon Elastic Container
+    Registry (ECR). You may then reference the URI of the newly built image.
     Its format will be as follows:
     
-    > **557821124784.dkr.ecr.us-east-1.amazonaws.com/evergreen/${directory}:${sha}**
+    > **557821124784.dkr.ecr.us-east-1.amazonaws.com/evergreen/<DIRECTORY\>:<SHA\>**
```

Comment:
Since I thought of it, I think Docusaurus has some weird behavior with regards to `<` and `>`. If you want the angle brackets to render correctly to work regardless of the Markdown flavor, you can either put them in tildes (\`<\`), or rely on HTML and use `&lt;` to render `<` and `&gt` to render `>`.